es-css-modules/index.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>es-css-modules - CSS in JS</title>
+    <link rel="stylesheet" href="bundle.css">
```

Comment:
`bundle.css` doesn't exist after running `gulp` (the path is `dist/button.css`).